submissions/jeffersonMormah/javaScipt2/script.js

```diff
@@ -0,0 +1,18 @@
+var a = 1
+var b = 123;
+var c = 7;
+var d = b - a;
+
+for (var a = 1; a < d; a++){
+  if ((a % 3) === 0) {
+      console.log("Software")
+  } else if ((a % 5) === 0){
+      console.log("Developer")
+  } else if ((a % 3) && (a % 5)) {
```

Comment:
This condition is suppose to run first. As it is, you won't be able to log `software developer` because the first two conditions will be true before the third statement.

Reply:
rectified